db/schema.rb

```diff
@@ -33,9 +33,9 @@
   end
 
   create_table "users", force: :cascade do |t|
-    t.string "uid"
     t.string "name"
     t.string "email"
+    t.string "password_digest"
```

Comment:
Will we also then use the normal `bcrypt` method for authentication/authorization within the app as well with this `password_digest`?

Reply:
I believe Jermaine dealt with this.